projects/ng-bulma-lib/src/lib/compenents/modal/bm-modal-card/modal-card.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'bm-modal-card',
+  template: `
+    <div class="modal-card">
+      <ng-content select="[modal-card-header]"></ng-content>
+      <section class="modal-card-body">
+        <ng-content></ng-content>
+      </section>
+      <ng-content select="[modal-card-footer]"></ng-content>
+    </div>
+  `
+})
+export class ModalCardComponent implements OnInit {
+  constructor() { }
```

Comment:
Remove constructor and ngOnInil as they do nothing. Also remove white spaces after ngOnInit